packages/core/src/layout/ItemCard/ItemCard.tsx

```diff
@@ -64,13 +71,22 @@ export const ItemCard = ({
         {tags?.map((tag, i) => (
           <Chip label={tag} key={`tag-${i}`} />
         ))}
-        <Typography variant="body2" paragraph className={classes.description}>
+        <Typography
+          variant="body2"
+          paragraph
+          className={`${classes.description} ${
+            tags && tags.length > 0 ? classes.withTags : ''
```

Comment:
`classnames` or `clsx` (see elsewhere in the repo for usages) are great for this kind of thing

Reply:
Choose `clsx` as it comes with typings out of the box. `@backstage/code` as both dependencies, maybe we should settle on one?